package.json

```diff
@@ -27,10 +27,10 @@
   },
   "peerDependencies": {
     "chai": "^3.0.0 || ^4.0.0",
-    "cheerio": "0.19.x || 0.20.x || 0.22.x || 1.0.0-rc.1",
-    "enzyme": "1.x || ^2.3.0",
-    "react": "^0.14.0 || ^15.0.0-0",
-    "react-dom": "^0.14.0 || ^15.0.0-0"
+    "cheerio": "0.19.x || 0.20.x || 0.22.x || ^1.0.0-0",
+    "enzyme": "^2.7.0 || ^3.0.0",
```

Comment:
this is semver-major since it's dropping v1, but i don't think it's worth retaining v1 support either.

Reply:
Good point. Since the current version is still < 1.0, does this make it 0.9.0?

Reply:
IMO we release a 1.0 when this PR is merged in

Reply:
1.0 is always good :-) if it remained sub-1.0 for some reason then it'd be `0.9.0`, yes